plugin/src/main/kotlin/com/jraska/module/graph/assertion/GradleDependencyGraphFactory.kt

```diff
@@ -30,7 +30,7 @@ object GradleDependencyGraphFactory {
   }
 
   private fun Project.listAllDependencies(configurationsToLook: Set<String>): List<Pair<String, List<String>>> {
-    return rootProject.subprojects
```

Comment:
I need to think a bit about possible edge cases and if this would cause breaking change

1. For users with not having any dependencies in root project - the module will be dropped
2. For users with single `:app` module who apply the plugin in root project, it might create a singular graph of one node with `:root` instead of `:app`, but actually this might be the case already. 

I need to basically sit on this once I get a moment to verify the edge cases 🤔 the major cases should be fine.

Reply:
I've realized that using `allprojects` instead of `subprojects` is changing plugin's behavior as the order of `allprojects` is different (some of the tests that were checking Gradle output started failing), so I've changed `allprojects` to `subprojects` + `rootProject`. It will make this change backward compatible, as the order of the graph nodes will be the same.

For possible edge cases I've created test `printsCorrectStatisticsForIndependentRootProject`. It gives the exact same results for `subprojects` and `subrojects` + `rootProject`, so there should be no effect on cases where `rootProject` has no dependencies.